common/flatpak-utils.c

```diff
@@ -616,6 +616,53 @@ flatpak_get_have_intel_gpu (void)
   return have_intel;
 }
 
+static GHashTable *
+load_kernel_module_list (void)
+{
+  GHashTable *modules = g_hash_table_new_full (g_str_hash, g_str_equal, g_free, NULL);
+  g_autofree char *modules_data = NULL;
+  g_autoptr(GError) error = NULL;
+  char *start, *end;
+  
+  if (!g_file_get_contents ("/proc/modules", &modules_data, NULL, &error))
+    {
+      g_debug ("Failed to read /proc/modules: %s", error->message);
+      return modules;
+    }
+
+  /* /proc/modules is a table of modules.
+   * Columns are split by spaces and rows by newlines.
+   * The first column is the name. */
+  start = modules_data;
+  while (TRUE)
+    {
+      end = strchr (start, ' ');
+      if (end == NULL)
+        break;
+
+      g_hash_table_add (modules, g_strndup (start, (end - start)));
+
+      start = strchr (end, '\n');
+      if (start == NULL)
+        break;
+
+      start++;
+    }
+
+  return modules;
+}
+
+static gboolean
+flatpak_get_have_kernel_module (const char *module_name)
+{
+  static GHashTable *kernel_modules;
```

Comment:
I guess static is defined to be zero initialized, but I would feel better with this initialized to NULL.